### ISSUE TITLE
Clean database entry when the creation of a new organization fails

### ISSUE THREAD

**Issue by llunaCreixent** [CirclesUBI/circles-myxogastria#221]
When the creation of a new organization process fails, the username is registered in the database locking its use forever. We could delete this entry when the Safe is not deployed so that the username can be reused. I wonder if the is any inconvenient if we "free" the username also when the Safe has been deployed but it hasn't signed up as an organization, leaving the Safe in an "invalid" state.

**Comment by adzialocha** [comment]
> I wonder if the is any inconvenient if we "free" the username also when the Safe has been deployed but it hasn't signed up as an organization, leaving the Safe in an "invalid" state.

Ja, it might be hard to find out "automatically" from the app perspective when something is considered "failed". Also giving the app the possibility to delete content automatically sounds problematic (except of when its behind some clear user consent UI).

Maybe we can have a worker task in `circles-api` which frequently goes through all username entries in the database and checks if there is an existing and functioning safe behind them?
